test/gmdifftest.c

```diff
@@ -24,58 +26,41 @@ static int check_time(long offset)
     int off_day, off_sec;
     long toffset;
     time_t t1, t2;
+
     time(&t1);
 
     t2 = t1 + offset;
     OPENSSL_gmtime(&t2, &tm2);
     OPENSSL_gmtime(&t1, &tm1);
     o1 = tm1;
     OPENSSL_gmtime_adj(&tm1, 0, offset);
-    if ((tm1.tm_year != tm2.tm_year) ||
-        (tm1.tm_mon != tm2.tm_mon) ||
-        (tm1.tm_mday != tm2.tm_mday) ||
-        (tm1.tm_hour != tm2.tm_hour) ||
-        (tm1.tm_min != tm2.tm_min) || (tm1.tm_sec != tm2.tm_sec)) {
-        fprintf(stderr, "TIME ERROR!!\n");
-        fprintf(stderr, "Time1: %d/%d/%d, %d:%02d:%02d\n",
-                tm2.tm_mday, tm2.tm_mon + 1, tm2.tm_year + 1900,
-                tm2.tm_hour, tm2.tm_min, tm2.tm_sec);
-        fprintf(stderr, "Time2: %d/%d/%d, %d:%02d:%02d\n",
-                tm1.tm_mday, tm1.tm_mon + 1, tm1.tm_year + 1900,
-                tm1.tm_hour, tm1.tm_min, tm1.tm_sec);
+    if (!TEST_int_eq(tm1.tm_year, tm2.tm_year) ||
+        !TEST_int_eq(tm1.tm_mon, tm2.tm_mon) ||
+        !TEST_int_eq(tm1.tm_mday, tm2.tm_mday) ||
+        !TEST_int_eq(tm1.tm_hour, tm2.tm_hour) ||
+        !TEST_int_eq(tm1.tm_min, tm2.tm_min) ||
+        !TEST_int_eq(tm1.tm_sec, tm2.tm_sec))
```

Comment:
Our coding style doesn't say so, but if you have looked at our code, you have probably seen that we prefer to break before operators rather than after.  I.e.:

``` C
    if (!TEST_int_eq(tm1.tm_year, tm2.tm_year)
￼        || !TEST_int_eq(tm1.tm_mon, tm2.tm_mon)
        || !TEST_int_eq(tm1.tm_mday, tm2.tm_mday)
        || !TEST_int_eq(tm1.tm_hour, tm2.tm_hour)
        || !TEST_int_eq(tm1.tm_min, tm2.tm_min)
        || !TEST_int_eq(tm1.tm_sec, tm2.tm_sec))
```

For consistency, would you mind...?